pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -400,6 +403,23 @@ class Build {
                             } else {
                                 additionalTestLabel += '&&sw.tool.podman'
                             }
+                        } else if (testType  == 'dev.jck') {
+                            jenkinsKey = 'Jenkins 4096 key'
+                            if (platform.contains("aix")) {
```

Comment:
I am not sure if you can use `platform` directly as it is not defined in this function.

Reply:
Switched to using buildConfig instead.